src/pages/ProfilePage.js

```diff
@@ -0,0 +1,103 @@
+import Header from "../components/Header";
+import userInfoStore from "../store/userInfoStore";
+import { navigate } from "../utils/navigate";
+
+const ProfilePage = () => {
+  const userInfo = userInfoStore.getUserInfo();
+  if (userInfo === null || userInfo === undefined) {
```

Comment:
초마이너) 요렇게 짧게도 쓸 수 있어유

```suggestion
  if (userInfo == null) {
```

Reply:
대박